papis/tui/widgets/list.py

```diff
@@ -240,15 +240,15 @@ def filter_options(self) -> None:
                     for i, opt in enumerate(self.options_matchers)
                     if i in search_indices
                 ]
+                results = [d.get() for d in results]
```

Comment:
Moving the `get()` inside the `with Pool..` is the actual fix to the problem I was having. It seems to have been incorrectly indented in #308.

Is the `get()` still necessary if the pool gets closed/joined after exiting the `with` block? (personal question as I'm not that familiar with the multiprocessing module)